kong/plugins/http-log/schema.lua

```diff
@@ -40,8 +41,30 @@ return {
               referenceable = true,
             },
           }},
+          { queue = typedefs.queue },
           { custom_fields_by_lua = typedefs.lua_code },
         },
+
+        entity_checks = {
+          { custom_entity_check = {
```

Comment:
Here we have duplicated entity check code, do we have some idea to eliminate it?

Reply:
What's the concern exactly?  I think we can live with this simple duplication as it will eventually be deleted anyway.